utils/sorting.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * @typedef {string} SortDirection
+ * @enum {SortDirection}
+ * @property {"asc"} ASC
+ * @property {"desc"} DESC
+ */
+
+/**
+ * Sorts repos by the a numeric value
+ * @param {UiRepo[]} repos
+ * @param {SortDirection} sortDirection
+ * @param {string} key - The key to sort by
+ * @returns {UiRepo[]}
+ */
+export const sortByNumericValue = (repos, sortDirection, key) => {
+  if (!sortDirection) return repos;
+  if (sortDirection === "asc") {
+    return repos.sort((a, b) => a[key] - b[key]);
+  }
+  return repos.sort((a, b) => b[key] - a[key]);
+};
+
+/**
+ * Sorts repositories by the date they were last updated
+ * @param {UiRepo[]} repos
+ * @param {SortDirection} sortDirection
+ * @returns {UiRepo[]}
+ */
+export const sortByUpdatedAt = (repos, sortDirection) => {
+  if (!sortDirection) return repos;
+
+  if (sortDirection === "asc") {
+    return repos.sort((a, b) => {
+      return a.updatedAtISO8601.localeCompare(b.updatedAtISO8601);
+    });
+  }
+  return repos.sort((a, b) => b.updatedAtISO8601 - a.updatedAtISO8601);
```

Comment:
Shouldn't this be `b.updatedAtISO8601.localeCompare(a.updatedAtISO8601)`?

I think having a test for this sort in descending order would be helpful.

Reply:
yeah good spot

Reply:
I've fixed this